From 1ee3824b30d9f415b685956494764a503711f86d Mon Sep 17 00:00:00 2001
From: mnesarka <quic_mnesarka@quicinc.com>
Date: Fri, 8 Mar 2024 11:39:27 +0530
Subject: [PATCH] meta-qti-auto: Remove queuing mechanism update to fq_codel

Fair Queue CoDel packet scheduler is not being used as queuing
mechanism for Linux networking. This patch removes queing mechanism
update to fq_codel.

Change-Id: I24d771e91d8fe523414687f3828afe20a80d53d1
---
 sysctl.d/50-default.conf | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/sysctl.d/50-default.conf b/sysctl.d/50-default.conf
index f41e24bcbc..b62dfe60fe 100644
--- a/sysctl.d/50-default.conf
+++ b/sysctl.d/50-default.conf
@@ -44,9 +44,6 @@ net.ipv4.conf.*.promote_secondaries = 1
 # systemd-based systems anyway: https://systemd.io/UIDS-GIDS#summary
 -net.ipv4.ping_group_range = 0 2147483647
 
-# Fair Queue CoDel packet scheduler to fight bufferbloat
--net.core.default_qdisc = fq_codel
-
 # Enable hard and soft link protection
 fs.protected_hardlinks = 1
 fs.protected_symlinks = 1
-- 
2.17.1

